test-gosysl.sh

```diff
@@ -8,3 +8,10 @@ for f in $ROOT/*.sysl; do
 done;
 
 rm $ROOT/*.out.txt
+
+ln -s $GOPATH/bin/sysl $GOPATH/bin/sd
+$GOPATH/bin/sd -a 'Project' $ROOT/sequence_diagram_project.sysl
+rm _.png
```

Comment:
How do you know if the file is getting generated here?
there is no server running on localhost:8080.
and the code here https://github.com/anz-bank/sysl/blob/master/sysl2/sysl/seqs/diagutil.go#L19 does not use value of SYSL_PLANTUML because plantuml has a default value. right?

Reply:
I revert the logic to check the SYSL_PLANTUML. If no value from command arg, will use the value from SYSL_PLANTUML. If env not setted, will use default value of "http://localhost:8080/plantuml".
And actually I'm not sure how to run the binary test. I add a new arg value for plantuml with "http://www.plantuml.com/plantuml". Hope it work.

Reply:
yep. I've removed the -p param from the go-testsysl.sh file.